gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -89,6 +90,9 @@
   /** Toolbar title, based on whether user is adding new or editing existing observation. */
   private final MutableLiveData<String> toolbarTitle = new MutableLiveData<>();
 
+  /** Captured / selected photo. */
+  private final MutableLiveData<Map<Field, String>> photoUpdated = new MutableLiveData<>();
```

Comment:
Please use an immutable map here to signal values can't be modified once in the stream.

Reply:
Done